cheatsheets/Choosing_and_Using_Security_Questions_Cheat_Sheet.md

```diff
@@ -1,119 +1,149 @@
+### Good Questions
 
-However, there is also some strong rationale to requiring users to create their own question(s), or at least one such question. The prevailing legal opinion seems to be if we provide some sort of reasonable guidance to users in creating their own questions and then insist on them doing so, at least some of the potential liabilities are transferred from our organizations to the users. In such cases, if user accounts get hacked because of their weak security questions (e.g., "What is my favorite ice cream flavor?", etc.) then the thought is that they only have themselves to blame and thus our organizations are less likely to get sued.
+Many good security questions are not applicable to all users, so the best approach is to give the user a list of security questions that they can choose from. This allows you to have more specific questions (with more secure answers), while still providing every user with questions that they can answer.
 
-Since OWASP recommends in the [Forgot Password Cheat Sheet](Forgot_Password_Cheat_Sheet.md) that multiple security questions should be posed to the user and successfully answered before allowing a password reset, a good practice might be to require the user to select 1 or 2 questions from a set of canned questions as well as to create (a different) one of their own and then require they answer one of their selected canned questions as well as their own question.
+The following list provides some examples of good questions:
 
-### Step 2) Review Any Canned Questions with Your Legal Department or Privacy Officer
+- What are the last four digits of your national insurance or social security number?
+- What was your maths teacher's surname in your 8th year of school?
+- What was the name of your first stuffed toy?
+- What was your driving instructor's first name?
 
-While most developers would generally first review any potential questions with whatever relevant business unit, it may not occur to them to review the questions with their legal department or chief privacy officer. However, this is advisable because their may be applicable laws or regulatory / compliance issues to which the questions must adhere. For example, in the telecommunications industry, the FCC's Customer Proprietary Network Information (CPNI) regulations prohibit asking customers security questions that involve "personal information", so questions such as "In what city were you born?" are generally not allowed.
+Much like passwords, there is a risk that users will re-use recovery questions between different sites, which could expose the users if the other site is compromised. As such, there are benefits to having unique security questions that are unlikely to be shared. An easy way to achieve this is to create more targeted questions based on the type of application. For example, on a share dealing platform, financial related questions such as "What is the first company you owned shares in?" could be used.
```

Comment:
I don't understand the examples much.
1. You discuss that on a virtual learning system, it's not good to have the 8th grade teacher.
2. On a sharing platform, it's good to say which share you owned?

Opposing points.
I do believe that questions can be made specific to certain platforms that forces the user to not re-use questions and answers, which is a positive thing.

Reply:
My point with the first one was that if you're still at school, asking a question about who your teachers are/were is bad because other people in the school are very likely to know that information.

I'll have a look at rephrasing this to make it more clear.

Reply:
Clarified the first point in e719853